tests/main.tf

```diff
@@ -41,13 +14,11 @@ module "lx-instance-centos7" {
   AmiDistro             = "CentOS"
   AppVolumeSize         = "10"
   CloudWatchAgentUrl    = var.cloudwatch_agent_url_centos
-  DisableRollback       = "true"
```

Comment:
Summarizing from the earlier discussion... 

The default here is `DisableRollback = false`. The default for `OnFailureAction` is `DO_NOTHING`. By removing both, we are inherently getting those default values instead.

I think at least _one_ of these configs should keep `DisableRollback = true`. That's the scenario we wanted to exercise, which previously required setting `OnFailureAction = ""`. It might be that we now have to set `OnFailureAction = null` instead.

Reply:
I confirmed that `OnFailureAction = null` works with `DisableRollback = true`.

From what I'm seeing, Terraform is verifying `OnFailureAction`.  It fails during `terraform plan`.  I think the conflict between the two parameters is verified in the Cloudformation API since in this case, it fails after `terraform apply` and when creation has been initiated.

I'm reverting things back but switching to `OnFailureAction = null`.